### PR TITLE
[zh] Translate projected volume page

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

**Comment by tengqm** [comment]
Preview: https://deploy-preview-31945--kubernetes-io-main-staging.netlify.app/zh/docs/concepts/storage/projected-volumes/

File: content/zh/docs/concepts/storage/projected-volumes.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+In general, granting the container access to the host is discouraged as it can
+open the door for potential security exploits.
+
+Creating a Windows Pod with `RunAsUser` in it's `SecurityContext` will result in
+the Pod being stuck at `ContainerCreating` forever. So it is advised to not use
+the Linux only `RunAsUser` option with Windows Pods.
+-->
+总体而言，为容器授予访问宿主系统的权限这种做法是不推荐的，因为这样做可能会打开潜在的安全性攻击之门。
+
+在创建 Windows Pod 时，如何在其 `SecurityContext` 中设置了 `RunAsUser`，
```
Comment (howieyuen) [inline content/zh/docs/concepts/storage/projected-volumes.md:222]:
```suggestion
在创建 Windows Pod 时，如果在其 `SecurityContext` 中设置了 `RunAsUser`，
```

**Review by howieyuen — CHANGES_REQUESTED** [review]

File: content/zh/docs/concepts/storage/projected-volumes.md [hunk 2]
```diff
@@ -0,0 +1,226 @@
+---
+title: 投射卷
+content_type: concept
+weight: 21 # just after persistent volumes
+---
+
+<!--
+reviewers:
```
[the remark this answers is not in the document]
Comment (tengqm) [reply content/zh/docs/concepts/storage/projected-volumes.md:8]: These lines are in comments. Our convention is to keep them.

[the remark this answers is not in the document]
Comment (howieyuen) [reply content/zh/docs/concepts/storage/projected-volumes.md:8]: last time, I was told to remove these, so I leave them not in comment? omg...

[the remark this answers is not in the document]
Comment (tengqm) [reply content/zh/docs/concepts/storage/projected-volumes.md:8]: probably. We are supposed to leave them their only as comments.

**Pull request merged**